plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/worker/KtLintWorkAction.kt

```diff
@@ -75,6 +81,13 @@ abstract class KtLintWorkAction : WorkAction<KtLintWorkAction.KtLintWorkParamete
             )
     }
 
+    private fun resetEditorconfigCache() {
+        if (parameters.editorconfigFilesWereChanged.get()) {
+            logger.info("Resetting KtLint caches")
+            KtLint.trimMemory()
```

Comment:
Can you add some documentation on why this is the method to call? `trimMemory` seems like a terrible method name for something like this, but maybe I'm misunderstanding something here.

Reply:
Basically this method call means to reduce KtLint internal memory caches that is also cleaning loaded `.editorconfig` cache. Will add a comment.